bpfassets/libbpf/src/kepler.bpf.c

```diff
@@ -127,14 +128,14 @@ static inline u64 calc_delta(u64 *prev_val, u64 *val)
 static inline u64 get_on_cpu_cycles(u32 *cpu_id)
 {
 	u64 delta, val, *prev_val;
-	struct bpf_perf_event_value c;
 	int error;
```

Comment:
nit: https://man7.org/linux/man-pages/man7/bpf-helpers.7.html#:~:text=long%20bpf_perf_event_read_value(struct%20bpf_map%20*map%2C%20u64%20flags%2C%0A%20%20%20%20%20%20%20struct%20bpf_perf_event_value%20*buf%2C%20u32%20buf_size) says - `long` instead of `int`